default/data/ui/views/setup_view_dashboard.xml

```diff
@@ -7,7 +7,8 @@ http://docs.splunk.com/Documentation/Splunk/latest/Viz/PanelreferenceforSimplifi
 -->
 <dashboard isDashboard="false"
            script="javascript/setup_page.js"
-           stylesheet="styles/setup_page.css">
+	       stylesheet="styles/setup_page.css"
```

Comment:
I think this may have accidentally inserted a tab (`\t`) instead of spaces

Reply:
Good catch, thanks!